database/migrations/sqlite/1717774945471180_all_team_locks.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS all_team_locks
+(
+    version BIGINT,
+    created TIMESTAMP,
+    environment VARCHAR(255),
+    teamName     VARCHAR(255),
+    json VARCHAR(255),
```

Comment:
I think for the teamName and env this is fine, but the json may be bigger than 255 characters.
```suggestion
    environment VARCHAR(255),
    teamName VARCHAR(255),
    json VARCHAR,
```